documentation/modules/configuring/proc-config-kafka-connect-s2i-migration.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='proc-migrating-kafka-connect-s2i-{context}']
+= Migrating from Kafka Connect with S2I to Kafka Connect
+
+[role="_abstract"]
+Support for Kafka Connect with S2I and the `KafkaConnectS2I` resource is planned for deprecation.
+The decision follows the introduction of `build` configuration properties to the `KafkaConnect` resource,
+which are used to build a container image with the connector plugins you require for your data connections automatically.
+
+This procedure describes how to migrate your Kafka Connect with S2I instance to a standard Kafka Connect instance.
+To do this, you configure a new `KafkaConnect` custom resource to replace the `KafkaConnectS2I` resource, which is deleted.
+
+WARNING: The migration process involves downtime from the moment `KafkaConnectS2I` instance is deleted until the new `KafkaConnect` instance has been successfully deployed.
+During this time, connectors will not be running and processing data. However, after the changeover they should continue from the point at which they stopped.
+
+.Prerequisites
+
+* Kafka Connect with S2I is deployed using a `KafkaConnectS2I` configuration
+* Kafka Connect with S2I is using an image with connectors added using an S2I build
+* Sink and source connector instances were created using `KafkaConnector` resources or the Kafka Connect REST API
+
+.Procedure
+
+. Create a new `KafkaConnect` custom resource using the same name as the name used for the `KafkaconnectS2I` resource.
+. Copy the `KafkaconnectS2I` resource properties to the `KafkaConnect` resource.
```

Comment:
Should they copy only the properties, or all of the YAML?

Reply:
It's pretty much just the kind that's different, so they could do either way and adjust accordingly.